### PR TITLE
Revamp README.md

### PR DESCRIPTION
**Pull request by QazCetelic**

### CODE REVIEW

**Comment by alex1701c** [comment]
Generally speaking, I wonder if there is even a point in repeating the instructions from the install.sh script in the readme.

What do you think about showing the clone command + command to execute the install script instead?

**Comment by QazCetelic** [comment]
Maybe it's better to leave the manual method out, the first option should suffice.

File: README.md [hunk 1]
```diff
@@ -1,59 +1,89 @@
...
+Option B: Get it from the [AUR](https://aur.archlinux.org/packages/plasma5-runners-emoji/)
 
-```
+<details>
+<summary>Option C: Manual method</summary>
+ 
+```bash
 git clone https://github.com/alex1701c/EmojiRunner
 cd EmojiRunner
 mkdir build
```
Comment (alex1701c) [inline README.md:72]: This is not needed when one uses the install script

File: README.md [hunk 2]
```diff
@@ -1,59 +1,89 @@
...
+```
 
-You can also find the plugin at the AUR: https://aur.archlinux.org/packages/plasma5-runners-emoji/
+</details>
 
-Or you can do it manually:
+Option B: Get it from the [AUR](https://aur.archlinux.org/packages/plasma5-runners-emoji/)
 
-```
+<details>
```
Comment (alex1701c) [inline README.md:66]: TBH I would not use the summary here, because the install instructions are pretty short and important information for readers

**Review by alex1701c — CHANGES_REQUESTED** [review]
Two neatpicks, then this is good to go :)

**Comment by QazCetelic** [comment]
Changed it, anything else?

File: README.md (hunk 1 above)
Comment (alex1701c) [reply README.md]: I only meant the `mkdir build` line, not the rest of the bash snippet

**Comment by alex1701c** [comment]
May I squash the commits when merging?

**Review by alex1701c — APPROVED** [review]

**Pull request merged**